README.md

```diff
@@ -81,13 +81,24 @@ The sections as enumerated below will walk you through the process of creating a
     node ./deploy.js -b <my-bucket> -s <my-stack> -i <my-docker-image-uri> -r <region>
     ```
    
-   Here is an example:
+    Here is an example:
     ```
     node ./deploy.js -b recording-demo-cnf-deploy-bucket -s recording-demo-cnf-stack -i 123456789012.dkr.ecr.us-east-1.amazonaws.com/recording-demo:latest -r us-east-1
     ```
-The above step deploys a AWS CloudFormation stack that creates resources needed to run the recording service. It may take several minutes to complete. You will get an Amazon API Gateway invoke URL in the output.
 
-![Deploy output](https://github.com/aws-samples/amazon-chime-sdk-recording-demo/blob/master/resources/deploy-script-output.png)
+    The above step deploys an AWS CloudFormation stack that creates resources needed to run the recording service. It may take several minutes to complete. You will get an Amazon API Gateway invoke URL in the output.
+    ![Deploy output](https://github.com/aws-samples/amazon-chime-sdk-recording-demo/blob/master/resources/deploy-script-output.png)
+
+    **Note:** If you encounter error messsage like: `IOError: [Errno 28] No space left on device.`, you can remove unused docker images/process by:
```

Comment:
Can we create an FAQ section in this readme and add this there? We can expand that section to add more such quick-fix steps.

Reply:
Yeah, will do.